src/RazorSdk/SourceGenerators/RazorSourceGenerationContext.cs

```diff
@@ -13,15 +13,15 @@ namespace Microsoft.NET.Sdk.Razor.SourceGenerators
 {
     internal class RazorSourceGenerationContext
     {
-        public string RootNamespace { get; private set; }
+        public string RootNamespace { get; private set; } = "ASP";
 
-        public IReadOnlyList<RazorInputItem> RazorFiles { get; private set; }
+        public IReadOnlyList<RazorInputItem> RazorFiles { get; private set; } = Array.Empty<RazorInputItem>();
 
-        public IReadOnlyList<RazorInputItem> CshtmlFiles { get; private set; }
+        public IReadOnlyList<RazorInputItem> CshtmlFiles { get; private set; } = Array.Empty<RazorInputItem>();
 
-        public VirtualRazorProjectFileSystem FileSystem { get; private set; }
+        public VirtualRazorProjectFileSystem? FileSystem { get; private set; }
```

Comment:
Can we use a constructor instead of Create if you want to avoid these patterns? These shouldn't be null in the ordinary case.

Reply:
The main issue is the early exit that occurs when we cannot resolve a `razorLanguageVersion` from the provided build properties. I updated this to throw an error if it is not defined but defaulted to `RazorLanguageVersion.Latest`. Thoughts?